CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ To better foster an open, innovative, and inclusive community, please refer to o
 
```

Comment:
@morning4coffe-dev (cc @jeromelaban)
Also, I think it would be best to do a sanity check on the external docs as well for UWP references on the different external repositories
https://github.com/unoplatform/uno/blob/1c0a37fab6f500b3b56745408d5f164a0d165e28/doc/import_external_docs.ps1#L8